openfisca_us/variables/fcc/acp/acp.py

```diff
@@ -0,0 +1,19 @@
+from openfisca_us.model_api import *
+
+
+class acp(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Affordable Connectivity Program amount"
+    documentation = "Affordable Connectivity Program amount"
+    definition_period = YEAR
+    unit = USD
+    reference = "https://www.fcc.gov/acp"
```

Comment:
Alternatively could reference the U.S. Code source: http://uscode.house.gov/view.xhtml?req=granuleid:USC-prelim-title47-section1752&num=0&edition=prelim

Reply:
thanks!